samples/DotNet/Microsoft.Azure.ServiceBus/BasicSendReceiveUsingQueueClient/readme.md

```diff
@@ -0,0 +1,162 @@
+# Get started sending and receiving messages from ServiceBus queues using QueueClient
+
+In order to run the sample in this directory, replace the following bracketed values in the `Program.cs` file.
+
+```csharp
+const string ServiceBusConnectionString = "{ServiceBus connection string}";
+const string QueueName = "{Queue path/name}";
+```
+
+Once you replace the above values run the following from a command prompt:
+   
+```
+dotnet restore
```

Comment:
Just read about it. You're right. `msbuild` didn't replace `dotnet`. It just wrapped around. Both work fine. (y)